rs_gazebo/launch/HQ.launch

```diff
@@ -1,11 +1,8 @@
 <launch>
-  
   <arg name="use_sim_time" default="true" />
   <arg name="gui" default="true" />
   <arg name="headless" default="false" />
-  <!-- <arg name="world_name" default="$(find rs_gazebo)/worlds/ss_hq4.world" /> -->
- <arg name="world_name" default="empty.world" /> -
-
+  <arg name="world_name" default= "$(find rs_gazebo)/worlds/$(arg world)" />
```

Comment:
Please specify the default argument world, otherwise, the launch file will be broken if arg world is not specififed

Reply:
Thanks, done